### PR TITLE
Added system and location information for catalog-info.yaml

### PR DESCRIPTION
**Pull request by rujutashinde**
Added system and location information so the Broker tool can be seen correctly under quality-community domain

### CODE REVIEW

**Comment by rujutashinde** [comment]
@JacobCallahan I put in a new PR instead of PR #254  we can close that one, so we are not waiting on updates there

File: catalog-info.yaml [hunk 1]
```diff
@@ -1,26 +1,11 @@
...
+  title: QE Broker Root Configuration
+  namespace: quality-community
+  description: A root collection of all QE Broker configurations for Backstage.
 spec:
-  type: library
-  owner: group:redhat/satellite-qe
-  lifecycle: preproduction
+  owner: group:redhat/quality-community
+  targets:
+    - ./catalog/system/broker.yaml
```
Comment (JacobCallahan) [inline catalog-info.yaml:11]:
```suggestion
    - ./catalog/system/broker.yaml

```

File: catalog/system/broker.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - ansible-tower
+    - aap
+    - docker
+    - podman
+  links:
+    - title: PyPi
+      url: https://pypi.org/project/broker
+spec:
+  owner: group:redhat/satellite-qe
+  domain: quality-community/quality-community
```
Comment (JacobCallahan) [inline catalog/system/broker.yaml:24]:
```suggestion
  domain: quality-community/quality-community

```

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
Missing some trailing newlines. Also, can you update your commit message to describe the changes, specifically why the additional file is used for metadata instead of the original?

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**